src/__test__/Country.test.js

```diff
@@ -0,0 +1,18 @@
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import renderer from 'react-test-renderer';
+import store from '../redux/store';
+import Country from '../components/Country';
+
```

Comment:
- [ ] It would be better if we fix the failing test cases as we have in the screenshot below :+1: 

![Screenshot from 2022-09-02 13-39-17](https://user-images.githubusercontent.com/46388113/188162518-75dc7cba-c303-4166-8cca-beddf89be939.png)